gson/src/main/java/com/google/gson/JsonArray.java

```diff
@@ -99,6 +99,61 @@ public boolean add(JsonElement element) {
     return elements.add(element);
   }
 
+  /**
+   * Adds the specified boolean to self in the specified position.
+   * All the element on the array will be moved to the right (increment index by one)
```

Comment:
```suggestion
   * All the elements on the array will be moved to the right (increment index by one).
```
Same for documentation of the other methods.

Additionally this is slightly misleading because not "all" elements will be moved to the right, instead something like the following might be more correct:
> The element which is already at that index (if any) and all subsequent elements will be moved to the right, i.e. their index is incremented by one.

Similar to the documentation of [`java.util.List.add(int, E)`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/List.html#add(int,E))